styles/styles-grid.css

```diff
@@ -0,0 +1,426 @@
+  grid-column: 5 / span 3;
+  grid-row: 2 / 3;
+  z-index: 2;
+}
+
+.date-sm {
+  display: none;
+}
+
+.check {
+  grid-column: 7 / span 5;
+  grid-row: 2 / 3;
+  text-align: center;
+}
+
+.check-sm {
+  display: none;
+}
+
+.check-sm-midl {
+  display: none;
+}
+
+.submit {
+  grid-column: 11 / span 3;
+  grid-row: 2 / 3;
+}
+
+label {
+  padding-left: 25px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 31px;
+  text-align: left;
+  color: var(--general-white);
+}
+
+input {
+  color: var(--text-color);
+  background: var(--general-white);
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 21px;
+  border: 3px solid #f3f3f4;
+  border-radius: 8px;
+}
+input[type="text"] {
+  padding-left: 24px;
+}
+
+#date {
+  border: 3px solid #f5bd41;
+  border-radius: 8px;
+}
+
+input[type="submit"] {
+  background: #3077c6;
+  border: 0;
+  font-weight: 500;
+  font-size: 24px;
+  line-height: 28px;
+  color: var(--general-white);
+}
+
+.app {
+  gap: 40px;
+
+}
+
+.app svg {
+  margin-bottom: 120px;
+}
+
+.app svg:first-child {
+  width: 135px;
+  height: 40px;
+  justify-self: end;
+}
+
+.app svg:last-child {
+  width: 120px;
+  height: 40px;
+  justify-self: start;
+}
+
+@media only screen and (max-width: 863px) {
+  /* For mobile phones: */
+
+  .sm-1-col {
+    grid-template-columns: 1fr;
+    grid-template-rows: auto;
+  }
+
+  .sm-2-col {
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: auto;
+  }
+
+  .sm-3-col {
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: auto;
+  }
+
+  .sm-4-col {
+    grid-template-columns: repeat(4, 1fr);
+    grid-template-rows: auto;
+  }
+
+  .sm-5-col {
+    grid-template-columns: repeat(5, 1fr);
+    grid-template-rows: auto;
+  }
+
+  .sm-6-col {
+    grid-template-columns: repeat(6, 1fr);
+    grid-template-rows: auto;
+  }
+
+  .header {
+    display: grid;
+    grid-template-columns: repeat(5, 1fr);
+    grid-template-areas: "title . mode login menu";
+    margin-bottom: 40px;
+  }
+
+  nav {
+    display: none;
+  }
+  .header--title {
+    width: 120px;
+    height: 24px;
+  }
+
+  .navigation--mode {
+    width: 16px;
+    height: 16px;
+  }
+
+  .navigation--login {
+    width: 24px;
+    height: 24px;
+  }
+
+  svg.menu {
+    grid-area: menu;
+    display: block;
+    width: 24px;
+    height: 16px;
+  }
+
+  .main-title {
+    font-size: 30px;
+    line-height: 35px;
+    margin-bottom: 0;
+  }
+
+  form.search {
+    width: 90%;
+    display: grid;
+    grid-template-columns: repeat(6, 1fr);
+    grid-template-rows: repeat(4, 1fr);
+    gap: 8px;
+    padding: 0;
+    border: 8px solid #f5bd41;
+    background-color: #f5bd41;
+    margin: 30px 16px;
+  }
+
+  label {
+    display: none;
+  }
+
+  input {
+    height: 64px;
+    border: none;
+  }
+
+  .name {
+    grid-column: span 6;
+    grid-row: 1 / 2;
+  }
+
+  .date {
```

Comment:
```suggestion
  .date {
    display:none;
  }
  
  .date-check-in {
    display: block;
```

Reply:
Research the case and use this technique for input `#check` replacement with 3 other inputs for mobile.